README.md

```diff
@@ -98,6 +98,42 @@ function visible() {
 
 When your element is near `300px` of the viewport, you get your callback / true result.
 
+### Specifying debounce value
+
+By default, debounce value for scroll and resize events is 15ms.
+
+You can change that with `debounce` param.
+
+```js
+var inViewport = require('in-viewport');
+var elem = document.getElementById('myFancyDiv');
+
+inViewport(elem, { debounce: 300 }, visible);
+
+function visible() {
+  alert('myfancyDiv is visible in the `customContainer`!');
+}
+```
```

Comment:
Currently, scroll/resize events are called every 15ms which isn’t that different than 0ms. If we give users option to change it, they can set it to some sensible value like 300ms. Good explanation of usecase can be see on [CSS-Tricks article](https://css-tricks.com/the-difference-between-throttling-and-debouncing/#article-header-id-2).